gstd/src/common/handlers.rs

```diff
@@ -25,43 +25,47 @@
 //! debug and non-debug mode, for programs built in `wasm32` architecture.
 //! For `debug` mode it provides more extensive logging.
 
-#[cfg(target_arch = "wasm32")]
-use {crate::ext, core::alloc::Layout, core::panic::PanicInfo};
+#[cfg(feature = "oom-handler")]
+use crate::ext;
 
-#[cfg(target_arch = "wasm32")]
+#[cfg(feature = "oom-handler")]
```

Comment:
here and below cfg attr seems better

Reply:
That would make `oom` exist without any features enabled; is that intentional?

Reply:
Why?

`#[cfg_attr(feature = "oom-handler", alloc_error_handler)]`

Reply:
Then `oom` function will still exist, although not annotated with `#[alloc_error_handler]`; but there is no need in keeping it there when compiling without OOM handling enabled